tensorflow/stream_executor/rocm/BUILD

```diff
@@ -274,6 +274,16 @@ cc_library(
     alwayslink = True,
 )
 
+cc_library(
+        name = "rocm_helpers",
```

Comment:
Hmmmm, I just find out that this review is in conflict with an existing PR I'm in work with for a while: #775. It looks like this review assumes `MIOpen` cannot do 3d pooling thus used local kernels to do pooling instead. However, that is not true any more.